src/style.css

```diff
@@ -1 +1,101 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+  font-weight: 400;
+  font-style: normal;
+}
+
+.body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Please ensure that you add a fallback font-family. While you have Arial and Helvetica as fallbacks, it's recommended to always include a generic family name such as 'sans-serif' at the end of your font stack.